service/src/test/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/CompositionMapperTest.java

```diff
@@ -1,32 +1,40 @@
 package uk.nhs.adaptors.oneoneone.cda.report.mapper;
 
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.hl7.fhir.dstu3.model.CarePlan;
 import org.hl7.fhir.dstu3.model.Coding;
 import org.hl7.fhir.dstu3.model.Composition;
 import org.hl7.fhir.dstu3.model.Encounter;
 import org.hl7.fhir.dstu3.model.EpisodeOfCare;
 import org.hl7.fhir.dstu3.model.Reference;
+import org.hl7.fhir.dstu3.model.ReferralRequest;
 import org.hl7.fhir.instance.model.api.IBaseResource;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.InjectMocks;
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
+
 import uk.nhs.connect.iucds.cda.ucr.CE;
 import uk.nhs.connect.iucds.cda.ucr.II;
 import uk.nhs.connect.iucds.cda.ucr.POCDMT000002UK01ClinicalDocument1;
 import uk.nhs.connect.iucds.cda.ucr.POCDMT000002UK01Component2;
 import uk.nhs.connect.iucds.cda.ucr.POCDMT000002UK01ParentDocument1;
 import uk.nhs.connect.iucds.cda.ucr.POCDMT000002UK01RelatedDocument1;
 
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
-
 @ExtendWith(MockitoExtension.class)
 public class CompositionMapperTest {
 
     private final IBaseResource episodeOfCareRefResource = new EpisodeOfCare();
+    private final CarePlan carePlan = new CarePlan();
+    private final List<CarePlan> carePlans = Arrays.asList(carePlan);
```

Comment:
That's a tricky thing. When you create an array using `Arrays.asList` passing in objects it will create a const size array. `list.add` will cause UnsupportedOperationException at runtime if you would like to add anything to that list.
If this list will be extended then use `new Array` or it always have one element then use `Collections.singletonList`

I know this is "only" test code, but it's good to remember about that even here to not make such mistake in prod code

Reply:
converted to `Collections.singletonList`